pandas/plotting/_core.py

```diff
@@ -2818,18 +2818,42 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw an area plot of the input series using matplotlib.
+
+        An area plot displays graphically quantitative data.
+        This function wraps the matplotlib area function.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional
+            Coordinates for X point.
+        y : label or position, optional
+            Coordinates for Y point.
+        kwds : optional
             Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
 
         Returns
         -------
-        axes : matplotlib.AxesSubplot or np.array of them
+        matplotlib.AxesSubplot or numpy.ndarray
+            Area plot generated.
+
+        See Also
+        --------
+        pandas.DataFrame.plot : Draw a plot using matplotlib.
+
+        Examples
+        --------
+        .. plot::
+            :context: close-figs
+
+            This example draws an area plot based on the length and width of
+            some animals, displayed in three bins
+
+            >>> df = pd.DataFrame({
+            ...     'sales': [3, 2, 3, 9, 10, 6],
+            ...     'visits': [20, 42, 28, 62, 81, 50],
+            ... })
+            >>> area = df.plot.area()
```

Comment:
I'm not sure about this, but if the result of this method is an axis object, perhaps it's better to assign it to a variable named `ax` like in other examples.

Can you add other examples where the usage of the `x` and `y` parameters is shown? Also an example where a `numpy.ndarray` is returned.

Reply:
Not sure about how to define a "usable" example, a little help would be appreciated!